src/vue/components/grammar.vue

```diff
@@ -50,6 +57,7 @@ import { Grammars } from 'alpheios-res-client'
 import Vue from '@vue-runtime'
 
 import BackIcon from '@/images/inline-icons/back.svg'
+import DefinitionsIcon from '@/images/inline-icons/definitions.svg'
```

Comment:
shouldn't this be the grammar icon instead?

Reply:
oh, I don't know why I mixed them up!

Reply:
Done 
![image](https://user-images.githubusercontent.com/12377640/71044215-cef44e00-217c-11ea-88e9-90280bbc3d15.png)